src/Arcus.Testing.Logging.NUnit/Extensions/ILoggerBuilderExtensions.cs

```diff
@@ -23,7 +23,9 @@ public static ILoggingBuilder AddNUnitTestLogging(this ILoggingBuilder builder,
             ArgumentNullException.ThrowIfNull(outputWriter);
 
             var logger = new NUnitTestLogger(outputWriter);
+#pragma warning disable CA2000 // Responsibility of disposing the created object is transferred to the caller
```

Comment:
Same minor remark regarding the Xml documentation.   On L15, xUnit is mentioned.  Should be NUnit I guess.

Reply:
Fixed in https://github.com/arcus-azure/arcus.testing/pull/454/commits/cb12992ce09c0cae0ac1cfc98896fa089da43bd3